amy/trainings/tests/test_training_progress.py

```diff
@@ -263,29 +263,15 @@ def test_progress_with_failed_status_requires_notes(self):
         with self.assertValidationErrors(["notes"]):
             failed_progress_no_notes.full_clean()
 
-    def test_form_invalid_if_trainee_has_no_training_task(self):
+    def test_form_valid_if_trainee_has_no_training_task(self):
+        """Regression test for https://github.com/carpentries/amy/issues/2440"""
         data = {
             "requirement": self.requirement.pk,
             "state": "p",
             "trainee": self.ironman.pk,
         }
         form = TrainingProgressForm(data)
-        self.assertEqual(form.is_valid(), False)
-        self.assertIn(
-            "not possible to add training progress", form.non_field_errors()[0]
-        )
-
-    def test_form_with_failed_status_requires_notes(self):
```

Comment:
Because this removal confused me when I looked back at the diff:
Despite the name, the content of this test content is a duplicate of `test_form_invalid_if_trainee_has_no_training_task`, and the intent is covered by `test_progress_with_failed_status_requires_notes` elsewhere in this file.